### PR TITLE
Create Online Git Page 

### PR DESCRIPTION
**Pull request by Rizwan-learn**
This pull request is to add a web page of Rizwan's portfolio as well as a live demo link.
Repo name to be changed to webpage link 
Webpage link is now  - https://rizwan-learn.github.io/

### CODE REVIEW

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @Rizwan-learn, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



 ### Required Changes ♻️

- [ ] **Kindly, provide a description of this pull request, explaining the improvements you bring to the project with this feature branch.**

- [ ] Please follow the [template of README from Microverse](https://github.com/microverseinc/readme-template);
Here you can see that the Live Demo section looks like this:
image
You can adapt it to look like this if you want:
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] Please make sure to follow the [template](https://github.com/microverseinc/readme-template) regarding all sections.
How your README looks like:
image

What your README sections should look like:
image

**Kindly make your README more similar to the examples above and you'll be ready for the next project**  😉


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Visual Studio and GitHub <br>
 GITHub Page <br>
 Figma Template <br>
 _________________
+# Live Demo 
+Live [Demo Link](https://rizwan-learn.github.io/)
```
Comment (elmar8287) [inline README.md:13]: - [ ] Well done 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Rizwan-learn.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 # Required Changes ♻️

- [ ] Well done with the implementation of all suggestions from the previous reviewer, but you still have to fix the PR description. You can edit it by pushing on 3 dots like on the screenshot:
Screen Shot 2022-02-21 at 21 21 21



## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @Rizwan-learn.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**